docs/versions/latest/Components/Archetypes/vRA 8.x/General/Known Issues.md

```diff
@@ -4,9 +4,7 @@ A list of known vRA 8.x archetype issues
 
 ## Table Of Contents
 
-1. [Custom Resources Known Issues](../Components/Custom%20Resources.md#known-issues)
```

Comment:
Why are we removing those 2 references?

Reply:
Hmmm..or they are not existing to begin with..

Reply:
Yeah 😄